docs/architecture/managers/sessions-manager.md

```diff
@@ -169,6 +174,14 @@ connections and try to create a new one.
 
 For further information, see [`ConnectionsManager`][connections_manager].
 
+#### Anycast<GetPeers>
+This message is periodically sent to itself ([`SessionsManager`][sessions_manager]).
```

Comment:
What for is this sent to itself? Please explain.

Reply:
Done!